EnableAzureArc.ps1

```diff
@@ -88,7 +88,7 @@ $tags = @{ # Tags to be added to the Arc servers
     Responsible = "Responsible"
 }
 
-$workfolder = "C:\temp"
+$workfolder = "$env:SystemDrive\temp"
```

Comment:
This directory might not exist, btw. 

Why do we prefer this over $env:TEMP

Reply:
We create the temp dir if it doesn't exist - Raul suggested that it is more accessible than env:temp since when running as system, env:temp is somewhere else